tests/Aspire.Dashboard.Tests/Integration/Playwright/PlaywrightFixture.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Playwright;
+using Polly;
+using Polly.Retry;
+using Xunit;
+
+namespace Aspire.Dashboard.Tests.Integration.Playwright;
+
+public class PlaywrightFixture : IAsyncLifetime
```

Comment:
https://playwright.dev/dotnet/docs/test-runners#xunit-support It might be better use one of the officially supported testing-libraries for playwright (MSTest or NUnit https://playwright.dev/dotnet/docs/test-runners#xunit-support)

Reply:
I would definitely prefer to use MSTest or NUnit, but I do want to balance that against other concerns that
- this would be the only test project not using xUnit, and
- there are other playwright tests using xUnit

@danmoseley Could we get your opinion on this?

Reply:
@mxschmitt whats your opinion here?

Reply:
> 
> > there are other playwright tests using xUnit
> 
> Is this referring to [that](https://github.com/dotnet/aspire/blob/521a2c63cbcc3dcbc5d93c99d8c6fbc3a0933b39/tests/Aspire.Workload.Tests/WorkloadTestsBase.cs#L26)?

Yes. And the Workload tests are using Playwright in (b) mode from your comment. We `PackageReference` `"Microsoft.Playwright"` nuget package, and use that.

Reply:
Yes, the concern is whether the benefits of switching to MSTest or nUnit outweigh the cost of supporting a singular test project not built with xUnit.

Reply:
> @danmoseley Could we get your opinion on this?

I don't have much to add: Aspire builds with reusable infrastructure from dotnet/arcade, which supports xunit, as the .NET platform repos picked xunit long ago. I do not see anything in there for mstest (as a test platform) or nunit so use of those would presumably have to be set up custom in this repo.

I defer to @radical/@joperezr

Reply:
I'm not sure what additional work would be needed on the CI to support that - on the build machine, and helix. Also, arcade provides many test attributes for xunit like `ActiveIssue` which we use to skip tests conditionally. We also have `RequiresDocker` which is xunit based too.

This would also mean additional documentation, and providing ways to have similar functionality for skipping tests etc.

IMHO, sticking to xunit would be better as it would utilize all the existing infrastructure.

Reply:
Sounds reasonable! As long as you provide a base class which gives them a browser context and test per test it should not be too bad of a compromise. Also make sure to add an expect method like [here](https://github.com/microsoft/playwright-dotnet/blob/86ad6c744350f97b01eba68dbb3b4c59efc9caf4/src/Playwright.MSTest/PlaywrightTest.cs#L129).